torch/csrc/distributed/rpc/rref_impl.h

```diff
@@ -290,6 +299,10 @@ class TORCH_API OwnerRRef final : public RRef {
     return true;
   }
 
+  inline bool isConfirmed() const override {
```

Comment:
Should we add a comment here that for OwnerRRef it's always confirmed to avoid confusion?